index.bs

```diff
@@ -357,8 +353,8 @@ This method takes the following parameters:
 - The <dfn>accountInformation</dfn> parameter specifies information about the user account for which the credential is being
     created. This is meant for later use by the authenticator when it needs to prompt the user to select a credential.
 
-- The <dfn>cryptoParameters</dfn> parameter supplies information about the desired properties of the credential to be created.
-    The sequence is ordered from most preferred to least preferred. The platform makes a best effort to create the most
+- The <dfn>supportedAlgorithms</dfn> and <dfn>supportedTypes</dfn> parameters supply information about the desired properties of the credential to be created.
+    The sequence is ordered from most preferred to least preferred. The platform makes a best effort to create the most logical and
```

Comment:
what is meant by "most logical" in regards to a credential and its associated key pair?